src/blog/2018-9-26/csharp-and-scala-support.pug

```diff
@@ -0,0 +1,57 @@
+extends ../../layout/blog-layout.pug
+block content
+    :markdown-it
+        ## Getting started
```

Comment:
The title for the blogpost is `Introducing Stryker.NET and Stryker4s, but the blog post starts with the new website. Maybe this is one other point for under: `Collaboration`?

Reply:
Agreed, I think starting with the new platforms makes more sense.

Reply:
Hmm ok, sounds good.